### PR TITLE
Implement iOS YAML devtools reader

### PR DESCRIPTION
**Pull request by ASV44**
Closes: #12

- [ ] Unit tests  
- [x] Changelog 
- [ ] Documentation (Readme) 

### What has been done
 1. Implement iOS YAML devtools reader

### How to test
1.  Open sample iOS app and ensure app builds successfully

### CODE REVIEW

**Comment by ASV44** [comment]
@maximbircu PR updated. Issue with building error was solved by new release of YAML iOS parser.

PR is ready to be reviewed and merged

**Comment by maximbircu** [comment]
Just noticed that we have:

iOS - `DevToolsConfiguration.swift` 
Android - `DevToolsConfigurationScreen.kt`

It would be nice to make them equal, considering that this is a part of public API.

There is no need to change in this PR; we can do it in the upcoming one where we'll implement the actual `DevToolsConfiguration` screen.

File: devtools/ios/DevtoolsFramework/Extensions/DevToolsExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+                       onConfigUpdate: @escaping (Bool) -> Void = { _ in }) -> DevTools {
+        let sources = KotlinArray(size: Int32(devToolsSource.count)) { index in
+            return devToolsSource[Int(truncating: index)]
+        }
+
+        return DevToolsCompanion().create(name: name, devToolsSource: sources) { isCriticalUpdate in
+            onConfigUpdate(Bool(truncating: isCriticalUpdate))
+        }
+    }
+}
```
Comment (maximbircu) [inline devtools/ios/DevtoolsFramework/Extensions/DevToolsExtensions.swift:15]: 👍

**Comment by maximbircu** [comment]
@ASV44 let's update the [CHANGELOG.md](https://github.com/maximbircu/devtools-library/blob/master/CHANGELOG.md), and we're good to merge the PR.
Btw - it would be nice to add https://github.com/maximbircu/devtools-library/pull/70 to the [CHANGELOG.md](https://github.com/maximbircu/devtools-library/blob/master/CHANGELOG.md) file as well as we forget to do this last time.

**Comment by ASV44** [comment]
@maximbircu Changelog updated.

**Review by maximbircu — APPROVED** [review]

**Pull request merged**